samples/apps/copilot-chat-app/webapi/Model/PluginAuthHeaders.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace SemanticKernel.Service.Plugins;
+
+/// /// <summary>
+/// Represents the authentication headers imported Open API Plugin Skills.
```

Comment:
OpenAPI or OpenAI?

Reply:
Open API -- these headers contain the authentication information to connect to skills imported from Swagger specs

Reply:
Renamed it to make it more clear